### PR TITLE
Double default rate-limit tokens

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* Doubles the default number of tokens-per minute to 120

### CODE REVIEW

**Comment by mikehelmick** [comment]
this is a blanket increase - we don't want to increase the rate of /api/verify requests

just increase the rate on admin api issue requests

**Comment by whaught** [comment]
> this is a blanket increase - we don't want to increase the rate of /api/verify requests
> 
> just increase the rate on admin api issue requests

Ah I misunderstood, I thought you were advocating a blanket increase.

**Comment by sethvargo** [comment]
120 is fine, but change this for _only_ the `adminapi` in the terraform envvars.

File: terraform/main.tf [hunk 1]
```diff
@@ -189,7 +189,7 @@ export CACHE_REDIS_HOST="${google_redis_instance.cache.host}"
 export CACHE_REDIS_PORT="${google_redis_instance.cache.port}"
 
 export RATE_LIMIT_TYPE="REDIS"
-export RATE_LIMIT_TOKENS="60"
+export RATE_LIMIT_TOKENS="120"
```
Comment (mikehelmick) [inline terraform/main.tf]: this still applies to all services though

**Review by mikehelmick — COMMENTED** [review]
/lgtm

**Comment by whaught** [comment]
/close

This change was made directly to the deployed services, we aren't going to check-in a new default

**Pull request closed without merging**